clio-util/src/main/scala/org/broadinstitute/clio/util/ApiConstants.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.clio.util
```

Comment:
Not sure this makes sense to get packaged into the `util` project, `transfer-model` is effectively the client<->server API module.